frontend/site/index.md

```diff
@@ -1,11 +1,22 @@
----
----
+<script setup>
+import InventoryBase from './.vitepress/theme/components/InventoryBase.vue';
+</script>
 
-Lors du cinquième comité interministériel de la transformation publique qui s’est tenu à Mont-de-Marsan le 5 février dernier, le Premier ministre s’est engagé à ce que le Gouvernement mène une politique d’ouverture de la donnée publique ambitieuse, visant à mieux exploiter, ouvrir, partager et valoriser les données publiques.
+# Bienvenue sur le tableau de suivi des ouvertures
 
-Par [une circulaire du 27 avril 2021](https://www.legifrance.gouv.fr/circulaire/id/45162?dateSignature=&init=true&page=1&query=*&searchField=ALL&tab_selection=circ), qui s’appuie sur [les préconisations du rapport du député Éric Bothorel](https://www.mission-open-data.fr/uploads/decidim/attachment/file/36/Mission_Bothorel_Rapport.pdf) qui lui avait été remis le 23 décembre dernier, le Premier ministre demande à l’ensemble du Gouvernement de faire de la politique de la donnée une de ses priorités dans ses relations avec tous ses partenaires, notamment les collectivités territoriales, les acteurs privés et la société civile. Il a chargé la ministre de la transformation et de la fonction publiques de suivre la mise en œuvre de cette politique de la donnée.
+## Qu’est-ce que c’est ?
 
-Ce tableau permet de suivre et de rendre compte de l’ouverture des nouveaux jeux de données, codes sources et API publics, pour l’ensemble des ministères, inscrite dans leurs feuilles de route, en application de l’engagement 6 du CITP du 5 février 2021.
+Ce tableau permet de suivre et de rendre compte de l’ouverture des jeux de données, algorithmes, codes sources et API publics.
+
+Cette liste non exhaustive, inclut : les engagements ministériels pris lors du [CITP du 5 février 2021](https://www.legifrance.gouv.fr/circulaire/id/45162?dateSignature=&init=true&page=1&query=*&searchField=ALL&tab_selection=circ), les engagements ministériels prévus dans leurs [feuilles de route](https://www.data.gouv.fr/fr/datasets/feuilles-de-route-ministerielles-sur-la-politique-de-la-donnee-des-algorithmes-et-des-codes-sources/), les [données à forte valeur définies par la Commission européenne](https://ec.europa.eu/info/law/better-regulation/have-your-say/initiatives/12111-Donnees-ouvertes-disponibilite-des-ensembles-de-donnees-publiques_fr) ainsi que les demandes issues de la société civile reçu par l’équipe d’[Etalab](https://www.etalab.gouv.fr/).
+
+## Suggérer une ouverture de données
+
+Vous pouvez nous adresser une suggestion de publication qui nous aideront à prioriser les efforts d’ouverture.
+
+Note : les demandes effectuées ici ne sont pas assimilables à une demande adressée à la [Commission d’Accès aux Documents Administratifs](https://www.cada.fr/).
+
+<a href="" class="fr-btn">Faire une demande d'ouverture</a>
 
 ## Inventaire
```

Comment:
We'll need to find a strategy to update the dataset on datagouv referenced in the next section though

Reply:
Manual for now : https://github.com/etalab/data.gouv.fr/issues/1098